spec.yaml

```diff
@@ -89,10 +89,16 @@ paths:
                 "code": "RESOURCE_CREATED",
                 "message": "A new cluster has been created with ID 'wqtlq'"
               }
-        default:
-          description: Error
+        "401":
+          description: Permission Denied
```

Comment:
I totally agree that we should document the expected responses. I simply don't want to put work into duplicating an error 500 description for every operation.